lib/cylc/cfgspec/suite.py

```diff
@@ -352,6 +354,16 @@ def coerce_interval_list(value, keys, args, back_comp_unit_factor=1,
                 'execution timeout'           : vdr( vtype='interval_minutes'),
                 'reset timer'                 : vdr( vtype='boolean', default=False ),
                 },
+            'events' : {
+                'handler'                     : vdr( vtype='string_list', default=[] ),
+                'handler events'              : vdr( vtype='string_list', default=[] ),
+                'handler retry delays'       : vdr( vtype='interval_minutes_list', default=[] ),
+                'mail events'                 : vdr( vtype='string_list', default=[] ),
+                'mail from'                   : vdr( vtype='string' ),
+                'mail retry delays'           : vdr( vtype='interval_minutes_list', default=[] ),
+                'mail smtp'                   : vdr( vtype='string' ),
+                'mail to'                     : vdr( vtype='string' ),
+                },
```

Comment:
Yes, in theory. However, this change is an add-on to the `[runtime][TASK][event hooks]` section, which does not currently have the equivalent of #1501, so I am a bit reluctant to bundle this in as part of this change. I agree that we should have a follow on to implement #1501 functionality for `[runtime][TASK][event hooks]` and `[runtime][TASK][events]`.